### ISSUE TITLE
GatsbyJS + React use simple API-key.

### ISSUE THREAD

**Issue by borodin-dev** [gatsbyjs/gatsby#6562]
Hi! 
I have an API-key from admin-panel, where I have created posts, pages, etc. 
And I want to use it with **GatsbyJS+React** to get data from admin-panel to my react-project, using **gatsby-source-api**, but I don't understand how exactly I can do it, documentation is very [poor](https://www.gatsbyjs.org/packages/gatsby-source-api/?=api). Or maybe you can advise me how can I do that. Also, I must note that this API-key is all I have.

![screenshot from 2018-07-19 16-30-18 1](https://user-images.githubusercontent.com/20676818/42945773-e62a9b62-8b71-11e8-8c8f-ad9e641c4167.png)

**Comment by brotzky** [comment]
Hey @borodin-dev. The documentation for that page is indeed bare.
Check this out as it's related and should lead you in the right direction:
https://www.gatsbyjs.org/tutorial/part-five/

**Comment by m-allanson** [comment]
The source to that plugin can be found at: https://github.com/dmiepub/gatsby-source-api

**Comment by borodin-dev** [comment]
@brotzky yes, i saw description by the link you gave, but unfortunatelly, when i'm install any plugin, and restart gatsby - graphql doesn't add anything new(_allFile_ or _file_ for **gatsby-source-filesystem**, for example). may be i'm doing smth wrong?

@m-allanson yes, it is, but how to use it i can't find some example or smth like this.

**Comment by borodin-dev** [comment]
@KyleAMathews maybe you can advise some kind of resolve for my problem, using gatsby + react?

**Comment by KyleAMathews** [comment]
@borodin-dev I'd try first going through the tutorial and then go from there!

**Comment by borodin-dev** [comment]
@KyleAMathews well, thanks anyway.
